verible/verilog/analysis/checkers/constraint-name-style-rule_test.cc

```diff
@@ -27,8 +27,12 @@ namespace analysis {
 namespace {
 
 using verible::LintTestCase;
+using verible::RunConfiguredLintTestCases;
 using verible::RunLintTestCases;
 
+// Lower snake case, starts with `c_`
+#define kPrefix "c+(_[a-z0-9]+)+"
```

Comment:
I'd probably move this into the `VariousPrefixTests` TEST to be close to where it is needed.
(and since it is only used in one place to assemble the `pattern:` configuration, maybe it doesn't need to
be a macro at all).

Reply:
(other than that, looks great and ready to be merged)

Reply:
I agree. I needed this because my original intent was to offer a choice between `_c` suffix and prefix. Thanks!